1.21.6 Unable to be used

It seems to be an anomaly related to the NBT API.？
Please help  Thank you！

[23:16:08 ERROR]: Could not pass event PlayerInteractEntityEvent to CatchBall v2.1.0
tw.maoyue.catchball.libs.nbtapi.NbtApiException: [?][2.15.1-SNAPSHOT]Exception while getting NBTCompound from NMS Entity!
        at CatchBall-2.1.0.jar/tw.maoyue.catchball.libs.nbtapi.NBTReflectionUtil.getEntityNBTTagCompound(NBTReflectionUtil.java:320) ~[CatchBall-2.1.0.jar:?]
        at CatchBall-2.1.0.jar/tw.maoyue.catchball.libs.nbtapi.NBTEntity.getCompound(NBTEntity.java:81) ~[CatchBall-2.1.0.jar:?]
        at CatchBall-2.1.0.jar/tw.maoyue.catchball.libs.nbtapi.NBTCompound.getResolvedObject(NBTCompound.java:102) ~[CatchBall-2.1.0.jar:?]
        at CatchBall-2.1.0.jar/tw.maoyue.catchball.libs.nbtapi.NBTReflectionUtil.getData(NBTReflectionUtil.java:771) ~[CatchBall-2.1.0.jar:?]
        at CatchBall-2.1.0.jar/tw.maoyue.catchball.libs.nbtapi.NBTCompound.getString(NBTCompound.java:198) ~[CatchBall-2.1.0.jar:?]
        at CatchBall-2.1.0.jar/com.github.nutt1101.utils.NBTHandler.isCustomEntity(NBTHandler.java:57) ~[CatchBall-2.1.0.jar:?]
        at CatchBall-2.1.0.jar/com.github.nutt1101.event.HitEvent.getIsCustomEntity(HitEvent.java:409) ~[CatchBall-2.1.0.jar:?]
        at CatchBall-2.1.0.jar/com.github.nutt1101.event.HitEvent.handleEntityCatch(HitEvent.java:222) ~[CatchBall-2.1.0.jar:?]
        at CatchBall-2.1.0.jar/com.github.nutt1101.event.HitEvent.onPlayerInteractEntity(HitEvent.java:153) ~[CatchBall-2.1.0.jar:?]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[paper-api-1.21.6-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:71) ~[paper-api-1.21.6-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54) ~[paper-1.21.6.jar:1.21.6-35-0caf75f]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:131) ~[paper-1.21.6.jar:1.21.6-35-0caf75f]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:628) ~[paper-api-1.21.6-R0.1-SNAPSHOT.jar:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl$1.performInteraction(ServerGamePacketListenerImpl.java:2763) ~[paper-1.21.6.jar:1.21.6-35-0caf75f]
        at net.minecraft.server.network.ServerGamePacketListenerImpl$1.onInteraction(ServerGamePacketListenerImpl.java:2815) ~[paper-1.21.6.jar:1.21.6-35-0caf75f]
        at net.minecraft.network.protocol.game.ServerboundInteractPacket$InteractionAction.dispatch(ServerboundInteractPacket.java:140) ~[paper-1.21.6.jar:1.21.6-35-0caf75f]
        at net.minecraft.network.protocol.game.ServerboundInteractPacket.dispatch(ServerboundInteractPacket.java:91) ~[paper-1.21.6.jar:1.21.6-35-0caf75f]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.handleInteract(ServerGamePacketListenerImpl.java:2753) ~[paper-1.21.6.jar:1.21.6-35-0caf75f]
        at net.minecraft.network.protocol.game.ServerboundInteractPacket.handle(ServerboundInteractPacket.java:78) ~[paper-1.21.6.jar:1.21.6-35-0caf75f]
        at net.minecraft.network.protocol.game.ServerboundInteractPacket.handle(ServerboundInteractPacket.java:14) ~[paper-1.21.6.jar:1.21.6-35-0caf75f]
        at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$0(PacketUtils.java:29) ~[paper-1.21.6.jar:1.21.6-35-0caf75f]
        at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.21.6.jar:1.21.6-35-0caf75f]
        at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:155) ~[paper-1.21.6.jar:1.21.6-35-0caf75f]
        at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[paper-1.21.6.jar:1.21.6-35-0caf75f]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1449) ~[paper-1.21.6.jar:1.21.6-35-0caf75f]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:176) ~[paper-1.21.6.jar:1.21.6-35-0caf75f]
        at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:129) ~[paper-1.21.6.jar:1.21.6-35-0caf75f]
        at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1429) ~[paper-1.21.6.jar:1.21.6-35-0caf75f]
        at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1423) ~[paper-1.21.6.jar:1.21.6-35-0caf75f]
        at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:139) ~[paper-1.21.6.jar:1.21.6-35-0caf75f]
        at net.minecraft.server.MinecraftServer.managedBlock(MinecraftServer.java:1380) ~[paper-1.21.6.jar:1.21.6-35-0caf75f]
        at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1388) ~[paper-1.21.6.jar:1.21.6-35-0caf75f]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1265) ~[paper-1.21.6.jar:1.21.6-35-0caf75f]
        at net.minecraft.server.MinecraftServer.lambda$spin$2(MinecraftServer.java:310) ~[paper-1.21.6.jar:1.21.6-35-0caf75f]
        at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
Caused by: tw.maoyue.catchball.libs.nbtapi.NbtApiException: [?][2.15.1-SNAPSHOT]Method not loaded! 'NMS_ENTITY_GET_NBT'
        at CatchBall-2.1.0.jar/tw.maoyue.catchball.libs.nbtapi.utils.nmsmappings.ReflectionMethod.run(ReflectionMethod.java:408) ~[CatchBall-2.1.0.jar:?]
        at CatchBall-2.1.0.jar/tw.maoyue.catchball.libs.nbtapi.NBTReflectionUtil.getEntityNBTTagCompound(NBTReflectionUtil.java:315) ~[CatchBall-2.1.0.jar:?]
        ... 35 more

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -3,7 +3,7 @@ plugins {
     `java-library`
     `maven-publish`
     kotlin("jvm") version "2.2.0"
-    id("com.gradleup.shadow") version "8.3.6"
+    id("com.gradleup.shadow") version "8.3.8"
     id("xyz.jpenilla.run-paper") version "2.3.1"
 }
 

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -60,7 +60,7 @@ repositories {
 dependencies {
     api("org.bstats:bstats-bukkit:3.1.0")
     api("com.jeff_media:SpigotUpdateChecker:3.0.4")
-    api("de.tr7zw:item-nbt-api:2.15.1-20250622.113959-6")
+    api("de.tr7zw:item-nbt-api:2.15.1")
     api("cn.handyplus.lib.adapter:FoliaLib:1.2.1")
     api("com.tchristofferson:ConfigUpdater:2.2-SNAPSHOT")
     compileOnly("org.spigotmc:spigot-api:1.20.5-R0.1-SNAPSHOT")

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -8,7 +8,7 @@ plugins {
 }
 
 group = "com.github.nutt1101"
-version = "2.1.0"
+version = "2.1.1"
 description = "CatchBall"
 java.sourceCompatibility = JavaVersion.VERSION_21
 

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -118,7 +118,7 @@ tasks.jar {
 }
 
 tasks.runServer {
-    minecraftVersion("1.21.5")
+    minecraftVersion("1.21.7")
 }
 
 runPaper.folia.registerTask()

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -60,7 +60,7 @@ repositories {
 dependencies {
     api("org.bstats:bstats-bukkit:3.1.0")
     api("com.jeff_media:SpigotUpdateChecker:3.0.4")
-    api("de.tr7zw:item-nbt-api:2.15.1-20250619.003022-5")
+    api("de.tr7zw:item-nbt-api:2.15.1-20250622.113959-6")
     api("cn.handyplus.lib.adapter:FoliaLib:1.2.0")
     api("com.tchristofferson:ConfigUpdater:2.2-SNAPSHOT")
     compileOnly("org.spigotmc:spigot-api:1.20.5-R0.1-SNAPSHOT")

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -2,7 +2,7 @@ plugins {
     java
     `java-library`
     `maven-publish`
-    kotlin("jvm") version "2.1.21"
+    kotlin("jvm") version "2.2.0"
     id("com.gradleup.shadow") version "8.3.6"
     id("xyz.jpenilla.run-paper") version "2.3.1"
 }

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -61,7 +61,7 @@ dependencies {
     api("org.bstats:bstats-bukkit:3.1.0")
     api("com.jeff_media:SpigotUpdateChecker:3.0.4")
     api("de.tr7zw:item-nbt-api:2.15.1-20250622.113959-6")
-    api("cn.handyplus.lib.adapter:FoliaLib:1.2.0")
+    api("cn.handyplus.lib.adapter:FoliaLib:1.2.1")
     api("com.tchristofferson:ConfigUpdater:2.2-SNAPSHOT")
     compileOnly("org.spigotmc:spigot-api:1.20.5-R0.1-SNAPSHOT")
     compileOnly("io.lumine:Mythic-Dist:5.9.0")

